roles/reproducer/tasks/devscripts_layout.yml

```diff
@@ -47,6 +47,7 @@
       local=/{{ _cluster }}.openstack.lab/
       domain={{ _cluster }}.openstack.lab
       address=/api.{{ _cluster }}.openstack.lab/{{ _api_addr }}
+      address=/api-int.{{ _cluster }}.openstack.lab/{{ _api_addr }}
```

Comment:
@psathyan - is this needed? (I think you said this is only required for Single-node-Openshift the other day?)

Reply:
Closing this - I guess it does not hurt.